packages/wallet/backend/src/app.ts

```diff
@@ -232,7 +237,7 @@ export class App {
     router.post('/wallet', isAuth, rapydController.createWallet)
     router.post('/updateProfile', isAuth, rapydController.updateProfile)
     router.post('/verify', isAuth, rapydController.verifyIdentity)
-    router.get('/rates/daily', rapydController.getRates)
+    router.get('/rates/daily', ratesController.getRates)
```

Comment:
What's the particular use case for this endpoint ?

Reply:
Is the frontend not using it? I thought we will have the frontend try to do the conversion beforehand, when the "receive" is toggled?

Reply:
got it out of the way, since frontend is using /rates -> the endpoint created for rafiki